submit-queue/github/github.go

```diff
@@ -308,16 +305,28 @@ func ForEachCandidatePRDo(client *github.Client, user, project string, fn PRFunc
 
 		glog.V(8).Infof("%v", issue.Labels)
 		if !HasLabels(issue.Labels, []string{"lgtm", "cla: yes"}) {
+			glog.V(2).Infof("Skipping %d - doesn't have requisite labels", *pr.Number)
 			continue
 		}
-		if !HasLabel(issue.Labels, config.WhitelistOverride) && !userSet.Has(*prs[ix].User.Login) {
-			glog.V(4).Infof("Dropping %d since %s isn't in whitelist and %s isn't present", *prs[ix].Number, *prs[ix].User.Login, config.WhitelistOverride)
-			if _, _, err := client.Issues.AddLabelsToIssue(user, project, *prs[ix].Number, []string{"needs-ok-to-merge"}); err != nil {
-				glog.Errorf("Failed to set 'needs-ok-to-merge' for %d", *prs[ix].Number)
+
+		pr, _, err = client.PullRequests.Get(user, project, *pr.Number)
```

Comment:
are you intentionally shadowing `pr` here?
I guess at least comment as to why you're doing this?